product_abc_classification_sale/views/product_views.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<odoo>
+    <record id="product_template_only_form_view" model="ir.ui.view">
+        <field name="model">product.template</field>
```

Comment:
why not add <field name="name">?

Reply:
It is not necessary, it will be computed by the system